lib/external/sentry.js

```diff
@@ -24,25 +110,7 @@ const init = (config) => {
   Sentry.init({
     dsn: `https://${config.key}@sentry.io/${config.project}`,
     beforeSend(event) {
-      if (event.request) {
-        // clear out all cookie values
-        if (event.request.headers && event.request.headers.cookie) {
-          const cookies = event.request.headers.cookie.split('; ')
-            .map((cookie) => cookie.slice(0, cookie.indexOf('=')))
-            .join(', ');
-
-          event.request.headers.cookie = cookies; // eslint-disable-line no-param-reassign
-        }
-
-        // clear out all request body data for sensitive endpoints (logging in, etc)
-        if (event.request.url) {
-          for (const endpoint of sensitiveEndpoints) {
-            if (event.request.url.match(endpoint)) {
-              event.request.data = null; // eslint-disable-line no-param-reassign
-            }
-          }
-        }
-      }
+      event = sanitizeEventRequest(event); // eslint-disable-line no-param-reassign
```

Comment:
could just `const sanitized =` rather than disabling eslint. if it's as simple as that to make the linter happy i usually do.